code/__HELPERS/unsorted.dm

```diff
@@ -1143,6 +1143,14 @@ GLOBAL_LIST_INIT(wallitems, typecacheof(list(
 			assembled += A
 	return assembled
 
+/atom/proc/Shake(pixelshiftx = 15, pixelshifty = 15, duration = 250) //Does a "shaking" effect on a sprite, code is from tgstation
```

Comment:
```suggestion
/atom/proc/Shake(pixelshiftx = 15, pixelshifty = 15, duration = 25 SECONDS) //Does a "shaking" effect on a sprite, code is from tgstation
```
This is measured in deciseconds, no?

Reply:
I wasn't sure if it should be measured in seconds or not, i'll change it